lib/babe/mocks/mock_state.go

```diff
@@ -0,0 +1,839 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: ./lib/babe/state.go
+
+// Package mock_babe is a generated GoMock package.
+package mocks
```

Comment:
`//go:generate` comment is missing I think.

Please also try to have it generated as `mock_state_test.go` in the `lib/babe` package so it doesn't pollute the Go API (tests exported things are not exported to other packages)